cyences_app_for_splunk/bin/user_inventory_v2_util.py

```diff
@@ -229,6 +241,8 @@ def get_user_details(self):
         return _users
 
     def _find_user(self, user_entry: UserEntry):
+        if is_user_ends_with(user_entry.user, user_ends_with=["$"], user_postfixes=self.user_postfixes):
```

Comment:
This function should only return boolean logically.

Reply:
Yes, it returns only boolean.